app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class PagesController < ApplicationController
+  def home; end
```

Comment:
if you have routes rails will find your view. So, if your action is empty you can remove it